packages/angular/test/base/e2e/src/standalone/tabs.spec.ts

```diff
@@ -1,21 +1,47 @@
 describe('Tabs', () => {
-  beforeEach(() => {
-    cy.visit('/standalone/tabs');
+  describe('Without IonRouterOutlet', () => {
+    beforeEach(() => {
+      cy.visit('/standalone/tabs');
+    });
+  
+    it('should redirect to the default tab', () => {
+      cy.get('app-tab-one').should('be.visible');
+      cy.contains('Tab 1');
+    });
+  
+    it('should render new content when switching tabs', () => {
+      cy.get('#tab-button-tab-two').click();
+      cy.get('app-tab-two').should('be.visible');
+      cy.contains('Tab 2');
+    });
+  
+    // Fixes https://github.com/ionic-team/ionic-framework/issues/28417
```

Comment:
~ The test doesn't technically fix the issue, but is covering the behavior reported in the linked issue.

```suggestion
    // Issue: https://github.com/ionic-team/ionic-framework/issues/28417
```